### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Serhii-Leonenko**

### CODE REVIEW

**Comment by Serhii-Leonenko** [comment]
image
image
image
image
image
image

File: static/css/styles.css [hunk 1]
```diff
@@ -1,3 +1,8 @@
 body {
     margin-top: 20px;
 }
+
+.sidebar-nav {
+    padding: 0;
+    list-style: none;
+}
```
Comment (rotsen18) [inline static/css/styles.css:8]: add newline

File: templates/includes/sidebar.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        <a href="{% url 'taxi:car-list' %}">
+        All cars
+        </a>
+    </li>
+    <li>
+        <a href="{% url 'taxi:driver-list' %}">
+        All drivers
+        </a>
+    </li>
+</ul>
```
Comment (rotsen18) [inline templates/includes/sidebar.html:22]: add newline

**Review by rotsen18 — CHANGES_REQUESTED** [review]

File: static/css/styles.css (hunk 1 above)
Comment (Serhii-Leonenko) [reply static/css/styles.css:8]: +

File: templates/includes/sidebar.html (hunk 2 above)
Comment (Serhii-Leonenko) [reply templates/includes/sidebar.html:22]: +

**Review by rotsen18 — APPROVED** [review]

**Review by MasterpieceElbow — APPROVED** [review]

**Pull request closed without merging**